README.md

```diff
@@ -67,16 +67,8 @@ These scripts are for downloading data from gharchive.org, processing and loadin
 
 1) Install MS SQL Server.
 
-2) Create a file secrets.py in the same directory as the other python files. A sample file is included.
-   
-   In the file create the follow fields:
-                   
-        UID = 'login        
-   
-        PWD = 'password'
-   
-        Server = 'server_name'
-
+2) Create a file `local.yml` (by default this file added to .gitignore) in the directory [`configs`](osci/config/files). 
+A sample file [`default.yml`](osci/config/files/default.yml) is included, please don't change values in this file
```

Comment:
A good security practice is provide the ability to people to not store passwords on file system.